### ISSUE TITLE
Importing Ed25519 Private Kay for Wallet

### ISSUE THREAD

**Issue by b00f** [pactus-project/pactus#1550]
Currently, the wallet can only import BLS private keys. With command-line options, Pactus Wallet should be able to parse and import Ed25519 private keys as well.

**Comment by b00f** [comment]
We need to add an option to the command line command.

**Comment by Ja7ad** [comment]
> We need to add an option to the command line command.

This not problem, similar this issue we have in #1549

**Comment by b00f** [comment]
Yes, the clean way is to get type of the private key from user. We can detect it but it is not a clean way.

**Comment by b00f** [comment]
We may be able to detect the type of private key, by decoding the key. It is possible.
